python/pyspark/sql/group.py

```diff
@@ -70,6 +70,10 @@ def __init__(self, jgd: JavaObject, df: DataFrame):
         self._df = df
         self.session: SparkSession = df.sparkSession
 
+    def __repr__(self) -> str:
+        j_str = self._jgd.toString()
+        return "GroupedData" + j_str[26:]
```

Comment:
qq what's this 26?

Reply:
it is used to truncate the string from JVM `RelationalGroupedDataset: [grouping expressions: [e: string], value: [e: string, f: string], type: GroupBy]`